.github/workflows/tests/dfx-action.yml

```diff
@@ -0,0 +1,11 @@
+on: [push]
```

Comment:
Please set up this test to run on push to master, and on pull requests, but only if action.yml or this test changes.